src/simsopt/util/permanent_magnet_helper_functions.py

```diff
@@ -332,37 +334,36 @@ def initialize_coils(config_flag, TEST_DIR, s, out_dir=''):
         order = 5
 
         # qa needs to be scaled to 0.1 T on-axis magnetic field strength
-        from simsopt.mhd.vmec import Vmec
-        vmec_file = 'wout_LandremanPaul2021_QA_lowres.nc'
-        total_current = Vmec(TEST_DIR / vmec_file).external_current() / (2 * s.nfp) / 7.131
+        total_current = 187500
         base_curves = create_equally_spaced_curves(ncoils, s.nfp, stellsym=True, R0=R0, R1=R1, order=order, numquadpoints=128)
         base_currents = [(Current(total_current / ncoils * 1e-5) * 1e5) for _ in range(ncoils-1)]
         total_current = Current(total_current)
         total_current.fix_all()
         base_currents += [total_current - sum(base_currents)]
         coils = coils_via_symmetries(base_curves, base_currents, s.nfp, True)
-        # fix all the coil shapes so only the currents are optimized
-        for i in range(ncoils):
-            base_curves[i].fix_all()
+    
+    # fix all the coil shapes so only the currents are optimized
+    for i in range(ncoils):
+        base_curves[i].fix_all()
 
     # Initialize the coil curves and save the data to vtk
     curves = [c.curve for c in coils]
     curves_to_vtk(curves, out_dir / "curves_init")
     return base_curves, curves, coils
 
 
-def calculate_on_axis_B(bs, s):
+def calculate_modB_on_major_radius(bs, s):
     """
-    Check the average, approximate, on-axis
-    magnetic field strength to make sure the
-    configuration is scaled correctly.
+    Check the average magnetic field strength along the major radius
+    to make sure the configuration is scaled correctly. For highly shaped
+    stellarators, this can deviate a bit from the on-axis B field strength.
 
     Args:
-        bs: MagneticField or BiotSavart class object.
-        s: plasma boundary surface.
+        bs (BiotSavart): MagneticField or BiotSavart class object.
+        s (Surface): plasma boundary surface.
```

Comment:
Specifically, state that s must be a SurfaceRZFourier - otherwise the `get_rc()` method used in this function would not exist.